packages/react-components/src/components/tests/MessageThread.test.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { MessageThread } from '../MessageThread';
+import { ChatMessage } from '../../types';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { mountWithLocalization } from './enzymeUtils';
+import { act } from 'react-dom/test-utils';
+import germanStrings from '../../localization/translated/de.json';

```

Comment:
I like the test! But this should probably be a pseudo language that is used exclusively for testing - this gives the flexibility to test unusual things like obtuse unicode characters